starter_code/src/movies.js

```diff
@@ -2,14 +2,106 @@
 
 // Iteration 1: Ordering by year - Order by year, ascending (in growing order)
 
+function orderByYear(arr) {
+    let sortedArray = arr.sort((a, b) => {
```

Comment:
its better to make a copy of array and work on that one so we dont modife the original array ,
like this : let copyArray = arr.slice()